### ISSUE TITLE
Issues with NumberField broken when using vaadin-archetype-application

### ISSUE THREAD

**Issue by appreciated** [vaadin/flow#6620]
## Description of the bug
The `NumberField` Component seems to be broken in Vaadin 14.0.7.

![grafik](https://user-images.githubusercontent.com/1729845/66209912-cbdadd00-e6b8-11e9-9534-54f903a9da74.png)

## Minimal reproducible example
Create a project using the `vaadin-archetype-application` archetype
```
public class MainView extends VerticalLayout {
    public MainView() {
        Button button = new Button("Click me", event -> Notification.show("Clicked!"));
        TextField placeholderField = new TextField();
        placeholderField.setPlaceholder("Placeholder");
        NumberField numberField = new NumberField("Years of expertise");
        add(button, placeholderField, numberField);
    }
}
```

## Expected behavior
The  `NumberField` should behave as expected

## Actual behavior
The  `NumberField` cannot be used

## Versions:
    - Vaadin 14.0.7 / Flow version 2.0.14
    - Java version 8 and 11
    - Browser version updated FF and Chrome

**Comment by appreciated** [comment]
I was only able to get the `vaadin-archetype-application` project running by replacing the `package-lock.json` with one from another working project. Why is the project not able to recover on it's own by executing a `clean install`? If the `package-lock.json` is deleted another invalid will be generated.

**Comment by lucastanzani** [comment]
I think this issue is similar / related to #https://github.com/vaadin/vaadin-text-field-flow/issues/234
I am not using vaadin-archetype-application but an application using the app-layout addon by @appreciated and Spring Boot.

**Comment by heruan** [comment]
See https://github.com/vaadin/vaadin-themable-mixin/issues/65 (fixed in [v1.5.1](https://github.com/vaadin/vaadin-themable-mixin/releases/tag/v1.5.1))

**Comment by appreciated** [comment]
Deleting:
* `node_modules`
* `target`
* `package-lock.json`

Followed by a `mvn clean install`

Fixed it
